safenode/src/bin/cli/files.rs

```diff
@@ -0,0 +1,112 @@
+// Copyright 2023 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use safenode::{
+    client::{Client, Files},
+    protocol::address::ChunkAddress,
+};
+
+use bytes::Bytes;
+use clap::Parser;
+use eyre::Result;
+use std::{fs, path::PathBuf};
+use tracing::info;
+use walkdir::WalkDir;
+use xor_name::XorName;
+
+#[derive(Parser, Debug)]
+pub enum FilesCmds {
+    Upload {
+        /// The location of the files to upload.
+        #[clap(name = "files-path")]
+        files_path: PathBuf,
```

Comment:
Is this the location of a directory? You can add this to clarify:
`#[clap(name = "files-path", value_name="DIRECTORY")]`

I would also suggest renaming the argument to `path`. The `files` part isn't strictly necessary, since it's already localised to a `files upload` command.

Reply:
Great, thanks. Will fix.